app/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,12 @@
     <string name="bottom_nav_faq_activity_main">F.A.Q.</string>
     <string name="bottom_nav_home_activity_main">Home</string>
     <string name="bottom_nav_chat_activity_main">Chat</string>
+    <string name="title_activity_auth">Authenticate</string>
```

Comment:
move this to more appropriate location.
if there is none, then create a section for activity titles